pkg/operator/controllers/banner/const.go

```diff
@@ -0,0 +1,9 @@
+package banner
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+const (
+	BannerName         = "openshift-aro-sre"
+	TextContactSupport = "Please ask your cluster administrator to contact Azure or Red Hat support. Your cluster's resource ID: %s"
```

Comment:
`Your cluster is experiencing issues. Please contact Azure Red Hat support . Cluster resource ID: %s`

Reply:
I would remove the space between "support" and the period. Also, I like the original phrasing, "Azure or Red Hat support".

Reply:
The message text was approved by Product Management. I changed it to the approved message.